### PR TITLE
fix: blazingtext tag and upgrade sparkml version

### PR DESCRIPTION
**Pull request by ChuyangDeng**
*Issue #, if available:*
`image_uris` should retrieve blazingtext:latest but instead blazingtext:1 is retrieved: https://github.com/awslabs/amazon-sagemaker-examples/issues/1392

Also spark-ml-serving is upgraded to 2.4

*Description of changes:*
- update blazingtext version from "1" to "latest"
- upgrade spark-ml-serving version to 2.4

*Testing done:*
`tox -e py38 -- tests/unit/sagemaker/image_uris`

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/image_uri_config/blazingtext.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "scope": ["inference", "training"],
     "versions": {
-        "1": {
```
Comment (iyerr3) [inline src/sagemaker/image_uri_config/blazingtext.json:4]: This should remain as `1` and we should add `latest` as an alias to `1`.  That's the policy for all 1P algorithms.

File: tests/unit/sagemaker/image_uris/test_algos.py [hunk 2]
```diff
@@ -177,7 +177,8 @@ def test_algo_uris(algo):
 
     for region in regions.regions():
         uri = image_uris.retrieve(algo, region)
-        assert expected_uris.algo_uri(algo, accounts[region], region) == uri
+        version = "latest" if algo == "blazingtext" else 1
```
Comment (metrizable) [inline tests/unit/sagemaker/image_uris/test_algos.py:180]: should be a `str`?

```suggestion
        version = "latest" if algo == "blazingtext" else "1"
```

Comment (iyerr3) [reply tests/unit/sagemaker/image_uris/test_algos.py:180]: To reiterate above point: nothing special in BlazingText wrt to versioning. It should be same as other 1P algos

**Comment by ChuyangDeng** [comment]
closing per #1917

**Pull request closed without merging**